chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -274,21 +245,6 @@ public CertifiedProductController(CertifiedProductUploadManager uploadManager,
         return mapCertifiedProductDetailsToBasic.apply(certifiedProduct);
     }
 
-    /**
-     * Get "basic" information for a listing which includes all details
-     * except for certification results and cqm results.
-     * @param year two-digit year (14 or 15)
-     * @param testingLab two-digit ATL code
-     * @param certBody two-digit ACB code
-     * @param vendorCode assigned developer code
-     * @param productCode user-defined product code
-     * @param versionCode user-defined version code
-     * @param icsCode two-digit ICS code
-     * @param addlSoftwareCode single-digit additional software code (0 or 1)
-     * @param certDateCode certified date code (YYMMDD format)
-     * @return the basic information about the listing identified by the unique CHPL ID specified
-     * @throws EntityRetrievalException if the listing cannot be found
-     */
     @SuppressWarnings({"checkstyle:parameternumber"})
     @ApiOperation(value = "Get all basic information for a specified certified product.  Does not include "
             + "the CQM results and certification results.",
```

Comment:
Should also mention that we're excluding measures. Or maybe try to make it generic and say something about "does not include elements that are in other "non-/details" endpoints"